ansible/roles/fanout/templates/sonic_deploy.j2

```diff
@@ -1,56 +1,329 @@
+    {% endfor %}
+    },
+
+    "BUFFER_PG": {
+    {% for alias in device_conn[inventory_hostname] %}
+        "{{ alias }}|0": {
+            "profile": "[BUFFER_PROFILE|ingress_lossy_profile]"
+        },
+        "{{ alias }}|3-4": {
+            "profile": "[BUFFER_PROFILE|pg_lossless_100000_300m_profile]"
+        }{% if not loop.last %},{% endif %}
+    {% endfor %}
+    },
+
+    "CABLE_LENGTH": {
+        "AZURE": {
+    {% for alias in device_conn[inventory_hostname] %}
+        "{{ alias }}": "300m"{% if not loop.last %},{% endif %}
+    {% endfor %}
+        }
+    },
+
+    "VERSIONS": {
+        "DATABASE": {
+            "VERSION": "version_1_0_1"
+        }
+    },
+
+    "PFC_WD": {
+        "GLOBAL": {
+            "POLL_INTERVAL": "200"
+        }
+    },
+
+    "WRED_PROFILE": {
+        "AZURE_LOSSLESS": {
+            "red_max_threshold": "2097152",
+            "red_drop_probability": "5",
+            "wred_green_enable": "true",
+            "ecn": "ecn_all",
+            "green_min_threshold": "250000",
+            "red_min_threshold": "1048576",
+            "wred_yellow_enable": "true",
+            "yellow_min_threshold": "1048576",
+            "green_max_threshold": "2097152",
+            "green_drop_probability": "5",
+            "yellow_max_threshold": "2097152",
+            "yellow_drop_probability": "5",
+            "wred_red_enable": "true"
+        }
+    },
+
+    "SCHEDULER": {
+        "scheduler.0": {
+            "type": "DWRR",
+            "weight": "14"
+        },
+        "scheduler.1": {
+            "type": "DWRR",
+            "weight": "15"
+        }
+    },
+
+    "BUFFER_POOL": {
+        "egress_lossless_pool": {
+            "type": "egress",
+            "mode": "static",
+            "size": "15982720"
+        },
+        "egress_lossy_pool": {
+            "type": "egress",
+            "mode": "dynamic",
+            "size": "9243812"
+        },
+        "ingress_lossless_pool": {
+            "xoff": "4194112",
+            "type": "ingress",
+            "mode": "dynamic",
+            "size": "10875072"
+        }
+    },
+
+    "BUFFER_PROFILE": {
+        "egress_lossless_profile": {
+            "static_th": "15982720",
+            "pool": "[BUFFER_POOL|egress_lossless_pool]",
+            "size": "1518"
+        },
+        "egress_lossy_profile": {
+            "dynamic_th": "3",
+            "pool": "[BUFFER_POOL|egress_lossy_pool]",
+            "size": "1518"
+        },
+        "ingress_lossy_profile": {
+            "dynamic_th": "3",
+            "pool": "[BUFFER_POOL|ingress_lossless_pool]",
+            "size": "0"
+        },
+        "pg_lossless_100000_300m_profile": {
+            "xon_offset": "2288",
+            "dynamic_th": "0",
+            "xon": "2288",
+            "xoff": "268736",
+            "pool": "[BUFFER_POOL|ingress_lossless_pool]",
+            "size": "1248"
+        }
+    },
+
+    "FEATURE": {
```

Comment:
Will feature table cause issue when the image is 201811 or 201911?

201811 doesn't have feature table support, 201911 might not know all services?

Reply:
Let me try it. My guess is such a case would be ignored, but let me try it out